From 040c7598f022449a52024d82fefb0685dc0aae90 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Mon, 24 Apr 2017 16:11:53 +0200
Subject: [PATCH 60/60] Disable smooth scrolling on Linux

Smooth scrolling (while using mousewheel or arrow keys) is a really
dumb idea. Text naturally appears as a blur while it moves, and the
animation takes that-many milliseconds to finish, so it's a time
waster too.

Only the fallback setting, which is used on Linux, is changed; on
Windows/MacOS, smooth scrolling remains controlled by a system
setting.
---
 ui/gfx/animation/animation.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ui/gfx/animation/animation.cc b/ui/gfx/animation/animation.cc
index 4bc1b248aa77..bc6e8671eb88 100644
--- a/ui/gfx/animation/animation.cc
+++ b/ui/gfx/animation/animation.cc
@@ -100,7 +100,7 @@ bool Animation::ShouldRenderRichAnimation() {
 // static
 bool Animation::ScrollAnimationsEnabledBySystem() {
   // Defined in platform specific files for Windows and OSX.
-  return true;
+  return false;
 }
 #endif
 
-- 
2.12.2

